src/devices/PiJuice/PiJuiceStatus.cs

```diff
@@ -120,7 +121,7 @@ public Temperature GetBatteryTemperature()
         {
             var response = _piJuice.ReadCommand(PiJuiceCommand.BatteryTemperature, 2);
 
-            return new Temperature((response[1] << 8) | response[0], TemperatureUnit.DegreeCelsius);
+            return new Temperature(BinaryPrimitives.ReadInt16LittleEndian(new Span<byte>(response)), TemperatureUnit.DegreeCelsius);
```

Comment:
you don't need new Span<byte>, this is implicit with byte[]
Same everywhere you've used it